website/docs/docs/dbt-support.md

```diff
@@ -51,4 +62,20 @@ Types of questions you should ask your Solutions Architect and Sales Director:
 - It is time for our contract renewal, what options do I have?
 
 
-When you need help writing SQL, reviewing the overall performance of your project, or want someone to actually help build your dbt project, check out our list of [dbt Preferred Consulting Providers](https://www.getdbt.com/ecosystem/) or our [Services](https://www.getdbt.com/dbt-labs/services/) page!
+For help writing SQL, reviewing the overall performance of your project, or want someone to actually help build your dbt project, refer to the following pages: 
+- List of [dbt Preferred Consulting Providers](https://www.getdbt.com/ecosystem/).
+- dbt Labs' [Services](https://www.getdbt.com/dbt-labs/services/).
+
+### Severity level
+
+Support tickets are assigned a severity level based on the impact of the issue to your business. The severity level determines the priority level of support you will receive. For specific ticket response time, Support does not have access to an Enterprise account's service level agreement (SLA). Please contact your legal team for a copy of your contract.
```

Comment:
do we want to state this? it makes it seem like support don't know the individual account's response to since we're saying we don't have access to their contract. i wouldn't want users to think we're not treating their we're not treating their ticket based on their specific sla. maybe:

>Support tickets are assigned a severity level based on the impact of the issue on your business. The severity level determines the priority level of support you will receive. For specific ticket response time based on your Enterprise account's service level agreement (SLA), please refer to your Enterprise contract.